Vagrantfile

```diff
@@ -225,10 +237,11 @@ def get_domains(config)
   subdomains = %w( www webgrind adminer mailcatcher browsersync info )
 
   subdomains.each do |domain|
-    domains << "#{domain}.#{config['name']}.dev"
+    domains << "#{domain}.#{config['name']}.local"
   end
 
-  domains << config['name']+".seravo.dev" #test https-domain-alias locally
+  domains << config['name']+".seravo.local" # test https-domain-alias locally
```

Comment:
I moved that for other branch, but we need to have HTTPS_DOMAIN_ALIAS defined so that users can test https domain alias in vagrant. Is this what you're asking for?